kratos/utilities/atomic_utilities.h

```diff
@@ -152,8 +178,15 @@ inline void AtomicSubMatrix(TMatrixType1& target, const TMatrixType2& value)
 template<class TDataType>
 inline void AtomicMult(TDataType& target, const TDataType& value)
 {
+#ifdef KRATOS_SMP_OPENMP
     #pragma omp atomic
     target *= value;
+#elif defined(KRATOS_SMP_CXX11)
+    AtomicRef<TDataType> at_ref{target};
```

Comment:
just for the sake of curiosity ... but why here not using *=?

Reply:
std::atomic ref doesn't have `operator*`